Library/Homebrew/cmd/update-report.rb

```diff
@@ -30,6 +30,8 @@ def update_report
         ohai "Homebrew has enabled anonymous aggregate user behaviour analytics"
         puts "Read the analytics documentation (and how to opt-out) here:"
         puts "  https://git.io/brew-analytics"
+        puts "Sleeping for 10 seconds. Press CTRL+C if you'd like to configure analytics"
```

Comment:
A slight alternative would be to keep the 10 second sleep (to allow people to digest the message) but change the messaging to say "Only the next run of the `brew` command will be tracked". Potentially having both is the best course of action, and I'm willing to update the pr if that seems reasonable to the core maintainers :)